content/blog/testthat-3-0-0/index.Rmd

```diff
@@ -0,0 +1,113 @@
+---
+output: hugodown::hugo_document
+
+slug: testthat-3-0-0
+title: testthat 3.0.0
+date: 2020-10-08
+author: Hadley Wickham
+description: >
+    testhat 3.0.0 brings a raft of major improvements including snapshot 
+    testing and parallel testing. It also includes a new "edition" that
+    allows you opt-in to a set of substantial improvements that are
+    not backward compatible.
+
+photo:
+  url: https://unsplash.com/photos/209FvE_57H8
+  author: NeONBRAND
+
+categories: [package] 
+tags: [testthat]
+---
+
+We're tickled pink to announce the release of [testthat](http://testthat.r-lib.org/) 3.0.0.
+testthat makes it easy to turn your existing informal tests into formal automated tests that you can rerun quickly and easily.
+testthat is the most popular unit-testing package for R, and is used by over 5,000 CRAN and Bioconductor packages.
+You can learn more about unit testing at <https://r-pkgs.org/tests.html>.
+
+You can install it from CRAN with:
+
+```{r, eval = FALSE}
+install.packages("testthat")
+```
+
+In this blog post, I want to discuss the two biggest changes: the introduction of the edition, which allows us to make breaking changes that you have to specifically opt-in to, and a new family of "snapshot" tests.
+I'll also give a quick round of the other major changes, but you might also want to look at the [release notes](%7B%20github_release%20%7D) for the full details.
+
+```{r setup}
+library(testthat)
+```
+
+## 3rd edition
+
+testthat 3.0.0 introduces the idea of an **edition**.
+An edition is a bundle of behaviours that you have to explicitly choose to use, allowing us to make backward incompatible changes.
+This is particularly important for testthat since it's used by a very large number of packages, and it includes a few design infelicities that we want to fix.
```

Comment:
Clarify that you're referencing "old edition" w/ design infelicities (I'm not sure what term you're using for that).